scripts/create_input_data_overview.py

```diff
@@ -47,7 +47,7 @@
     df = load_b3_scalars(in_path)
 
     # filter for data within the scenario defined above
-    df = df.loc[df["scenario"] == SCENARIO]
+    df = df.loc[df["scenario_key"] == SCENARIO]
```

Comment:
```suggestion
    df = df.loc[df["scenario_key"] == SCENARIO_KEY]
```

Reply:
Need to change on top of the file as well.

Reply:
Done in [f4e5875](https://github.com/rl-institut/oemof-B3/pull/147/commits/f4e587503f4f2bc80e7def0f0924bb30a7fb5f50)